contact/async.js

```diff
@@ -18,11 +19,7 @@ exports.create = function (api) {
   })
 
   function followerOf (source, dest, cb) {
-    var following = api.contact.obs.following(source)
-    onceTrue(following.sync, () => {
-      var value = resolve(following)
-      cb(null, value && value.has(dest))
-    })
+    api.sbot.async.friendsGet({source: source, dest: dest}, cb)
```

Comment:
@dominictarr I'm trusting this returns a boolean when both source and dest are declare:

(folowed upstream to https://github.com/ssbc/ssb-friends/blob/master/index.js#L171-L182)

cc @ahdinosaur @mmckegg - Dominic found a bug where followerOf wasn't working with an empty feed.
merging this for that project we're building. We're open to reworking this if you'd prefer another solution